operator/pkg/reconciliation/reconcile_racks.go

```diff
@@ -1682,6 +1710,8 @@ func (rc *ReconciliationContext) ReconcileAllRacks() (reconcile.Result, error) {
 		return recResult.Output()
 	}
 
+	// TODO: Set condition Ready
+	
 	if err := setOperatorProgressStatus(rc, api.ProgressReady); err != nil {
 		return result.Error(err).Output()
 	}
```

Comment:
This file is getting uncomfortably large.  In a different ticket, we should consider splitting it up a bit.

Reply:
I don't think the size of the file is the real problem. It's the manner in which the functions are all interconnected.